src/styles.ts

```diff
@@ -5,64 +5,101 @@ export const styles = css`
 :host {
     font-family: var(--font-body);
     display: flex;
-    flex-flow: column;
-    align-items: center;
-}
-
-.topbar {
-    height: 66px;
-    width: 1025px;
-    background-color: #2C3D4E;
+    flex-flow: row;
+    margin: 0 auto;
+    justify-content: center;
 }
 
 .main-section {
     display: flex;
     justify-content: center;
     height: 100%;
-    height: 560px;
+    flex-wrap: wrap;
 }
 
-h2 {
+.form-template {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+.form-template > div {
+    width: 100%;
+}
+
+h1,
+h2,
+h3,
+h4,
+p,
+a,
+input {
+    font-family: 'Bariol', sans-serif;
```

Comment:
`var(--font-body)` is declared in the typography styles and would give you our standard chain of font fallbacks